.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv3/bin/python3"
+}
```

Comment:
missing newline

Reply:
Ah sorry. It was not my intention to commit any local file of my VSCode config. I removed it and added a directive to `.gitignore`.